pinot-core/src/test/java/org/apache/pinot/core/plan/maker/MetadataAndDictionaryAggregationPlanMakerTest.java

```diff
@@ -94,6 +94,7 @@ public void buildSegment()
     segmentGeneratorConfig.setTableName("testTable");
     segmentGeneratorConfig.setSegmentName(SEGMENT_NAME);
     segmentGeneratorConfig.setOutDir(INDEX_DIR.getAbsolutePath());
+    segmentGeneratorConfig.setCheckTimeColumnValidityDuringGeneration(false);
```

Comment:
maybe add a comment in these places as to the reason we are not checking for time column validity. I believe the reason is that the input avro files have invalid values for time column.

Reply:
Good point. Done